recipes/metasv/meta.yaml

```diff
@@ -1,32 +1,31 @@
 package:
   name: metasv
-  version: '0.4.0'
-
+  version: '0.5.4'
 source:
-  #fn: metasv-0.3.2a0.tar.gz
-  #url: https://github.com/bioinform/metasv/archive/0.3.2a0.tar.gz
-
-  fn: metasv-e499f09.tar.gz
-  url: https://github.com/bioinform/metasv/archive/e499f09.tar.gz
+  fn: 0.5.4.tar.gz
+  url: https://github.com/bioinform/metasv/archive/0.5.4.tar.gz
+  md5: de2e21ac4f86bc4d1830bdfff95d8391
 
 build:
-  number: 4
+  number: 0
   skip: True # [not py27]
 
 requirements:
   build:
     - python
     - setuptools
     - cython
-    - pybedtools
-    - pyvcf
-    - pysam
+    - pybedtools ==0.6.9
```

Comment:
Is this strict pinning really needed?

Reply:
For the versions, I'm following what the authors say to use under "System Requirements" here:  http://bioinform.github.io/metasv.